Source/Core/Core/HLE/HLE.cpp

```diff
@@ -185,13 +185,12 @@ bool IsEnabled(int flags)
 
 u32 UnPatch(const std::string& patch_name)
 {
-  auto* patch = std::find_if(std::begin(OSPatches), std::end(OSPatches), [&](const SPatch& patch) {
-    return patch_name == patch.m_szPatchName;
-  });
+  auto* patch = std::find_if(std::begin(OSPatches), std::end(OSPatches),
```

Comment:
nitpick, but i think range-based loop would be better here: capturing lambdas have overhead, and you wouldn't have to write as much repetitive stuff for begin/end.